UPDATING.md

```diff
@@ -1047,6 +1048,21 @@ If the DAG relies on tasks with other trigger rules (i.e. `all_done`) being skip
 
 The goal of this change is to achieve a more consistent and configurale cascading behaviour based on the `BaseBranchOperator` (see [AIRFLOW-2923](https://jira.apache.org/jira/browse/AIRFLOW-2923) and [AIRFLOW-1784](https://jira.apache.org/jira/browse/AIRFLOW-1784)).
 
+## Airflow 2.0
```

Comment:
It's actually Airflow Master :)

Reply:
BTW, should we put the latest change in the top of UPDATING.md @potiuk

Reply:
@potiuk Got it! Thanks!